programs/sig-aggregator/docker-compose.yml

```diff
@@ -0,0 +1,150 @@
+services:
```

Comment:
This _feels_ a bit much to just spin up these. Do we really need all the volumes and healthchecks and environments and all this? And there is a lot of inline scripts that are going to be a pain to maintain with hardcoded values everywhere.
For right now, it's OK to have a simple docker-compose for local testing, but I don't want to maintain that over time. As soon as we start integrating this into e2e stuff it should be spun up with code using something like testcontainers.

So I would say, try to simplify this.